src/instance.ts

```diff
@@ -136,6 +136,23 @@ export type SetInstanceMetadataCallback = (
 ) => void;
 export type SetInstanceMetadataResponse = [google.protobuf.Empty];
 
+export interface RestoreTableOptions {
+  gaxOptions?: CallOptions;
+}
+export type RestoreTableCallback = (
+  err: ServiceError | null,
+  apiResponse?: LROperation<
+    google.bigtable.admin.v2.ITable,
+    google.bigtable.admin.v2.IRestoreTableMetadata
+  >
+) => void;
```

Comment:
Long running operation functions that return a resource object usually have the return signatures as in this example

https://github.com/googleapis/nodejs-bigtable/blob/5e47bbc3d1304fb06cd2e6df5ff17c2a197a1352/src/instance.ts#L102-L111